packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/Confirm/template.success.tsx

```diff
@@ -47,6 +47,18 @@ const Success = (props: Props) => {
     ? (intervalToDuration({ end: props.withdrawLockCheck?.lockTime || 0, start: 0 }).days as number)
     : 0
 
+  const targetCoinName = useMemo(() => {
+    const currency = props.defaultMethod?.currency
+
+    if (!currency) return null
+
+    const coin = window.coins[currency]
+
+    if (!coin) return currency
+
+    return coin.coinfig.name
+  }, [props.defaultMethod])
```

Comment:
I've an underlying concern about this one. If this is an object, my understanding is that `useMemo` will be recalculated each time. I'd ditch the `useMemo` unless some really heavy lifting is being saved by it